src/codeu/chat/client/core/ConversationContext.java

```diff
@@ -32,19 +32,29 @@
   public final ConversationHeader conversation;
 
   private final BasicView view;
-  private final BasicController controller;
+  private final ClientController controller;
 
   public ConversationContext(User user,
                              ConversationHeader conversation,
                              BasicView view,
-                             BasicController controller) {
+                             ClientController controller) {
 
     this.user = user;
     this.conversation = conversation;
     this.view = view;
     this.controller = controller;
   }
 
+  // access = "member", "owner", "remove"
+  // return if ok
+  public boolean changeAccess(Uuid requestor, String userName, String access, Uuid conversation) {
```

Comment:
If access really only has three allowed values, you should probably make it an enum.